### DISCUSSION TITLE
AI助手讓你成為編輯高手：用最少的時間做出最好的影片

### DISCUSSION THREAD

**Discussion by JohnnyXiao1024** [1101-GS4719-C/Python-Community-GS4719-C#1207] [General]
相信各位在使用各種影音平台時
常常會被演算法推到一些大陸介紹影片
內容通常是電影介紹或是一些有的沒的

剛剛我嘗試使用chatGPT進行主題與腳本的生成
（特地選了自己完全不懂的主題，跟各位證明腳本都是AI操刀）
再搭配一些AI文生圖、AI配音的剪輯軟體，嘗試製作類似影片
還特地使用了簡體字幕，讓影片更有感wwww

影片連結：https://youtu.be/3nJ-4P1WN0I

![切开基因密码：基因编辑技术带来的新生命革命-封面](https://user-images.githubusercontent.com/125842539/229734642-fd27b22c-6b74-49f0-a281-cb1669a5f8a8.jpg)

**Comment by JohnnyXiao1024** [comment]
用不到三分鐘的製作時間，就能騙到大量觀眾的注意力
想到自己的時間也曾經被這樣偷走，一整個後悔莫及啊QQ

**Reply by txs** [reply]
很棒的是你從 YouTube 的觀看者變成了生產者，雖然只是花三分鐘，卻是關係的轉換的一大步。
應該滿多人對於基因編輯滿有興趣的，但是我對於騙到大眾的注意力這件事情是存疑的。
騙也許可以大量的得到短暫且顯著的效果，但是卻是耗費自身的信用，更可怕的是自己的認知。
真正的價值來自於真誠與累積，也許剛開始的成果不顯著，但持續的累積與成長才有良好的長期發展。
你得到大量的注意力，僅是因為尚未有人做出此類的影片，這是你在 YouTube 上的貢獻，而非騙的結果。

**Reply by txs** [reply]
這邊有一篇 UC Berkeley Professor Brian Harvey 的分享!
有時候問題點並非在事實，而是在認知。

https://youtu.be/hMloyp6NI4E

他的結論是作弊跟自殘沒兩樣

**Comment by txs** [comment]
這邊跟你分享一下，我做影片的一些心得，完全用 ChatGPT 與剪映圖文生成，我覺得也滿不錯的。我也有生成過這樣的內容測試，觀看數也不少。
我個人的頻道目前沒有使用完全的 AI 生成是因為我認為剪映的圖文生成功能還沒有強大到可以讓 80% 的人滿意，應該目前就 50% 的人覺得 OK 但應該不太會訂閱，除非是專業領域的文章用說的跟大家分享的那種頻道。
我個人還是使用自己錄素材, 簡單的影音處理，配字幕，塞梗圖，但花費滿大量的時間。
我以以下的影片作為例子
素材： 30 分鐘的錄影
去空後變為 12 分鐘的影片
加入字幕與梗圖後形成約 17 分鐘的影片。
剪輯總共花了 4 個小時，剪輯期間剪映會爆記憶體，所以需要多次重開
輸出時間若沒有爆記憶體大約 40 分鐘，爆記憶體約為一個半小時 (時間沒有精確計算)
我的觀點是不要拿自己投入的時間作為貢獻的依據，而是應該用影片提供的觀點與想法作為價值依據。
生命很可貴，所以可以將影片盡量拍短的價值反而會比長片的價值高。
剪片花費我大量的時間，有很多的流程我覺得如果可以用程式或是 AI 自動化解決應該會變得更棒。
歡迎多多交流 😃
https://www.youtube.com/watch?v=f4Ii4-G-JDM

**Comment by DuxtinTai** [comment]
感謝你們的分享！使用 AI 和圖文生成器創作影片，確實節省了大量的時間和成本，尤其是對於那些創作的頻率較高的內容創作者。不過，您也提到了 AI 生成的內容仍然無法滿足大多數人的需求，這點我完全同意您的觀點，價值應該是基於內容所能提供的觀點、想法和經驗，而非創作過程所耗費的時間。
將影片盡量拍短的想法也很值得，這樣可以讓觀眾更容易消化並吸收內容。當觀看影片時，我也喜歡看到精簡扼要的內容，尤其是當我希望快速了解某個主題或是問題。
